docs/topics/conditional-view-processing.txt

```diff
@@ -65,6 +65,9 @@ order, as the view function they are helping to wrap. The function passed
 last time the resource was modified, or ``None`` if the resource doesn't
 exist. The function passed to the ``etag`` decorator should return a string
 representing the `ETag`_ for the resource, or ``None`` if it doesn't exist.
+Additionally, it will set the ``ETag`` and ``Last-Modified`` headers on the
```

Comment:
Maybe a separate paragraph and some rephrasing would improve clarity. The new sentence follows one that starts with "The function passed to the etag decorator" so "it" could be interpreted as referring to that.